### DISCUSSION TITLE
if refined iron should only be made from a furnace and tech reborn furnace

### DISCUSSION THREAD

**Discussion by lunareclispe** [TechReborn/TechReborn#3129] [Ideas]
I was playing a modpack I made and I realized that refined iron had the same recipe as the other item I needed from the blast furnace so I think refined iron should only be made from a furnace (I know you can change the recipe but I don't know how and I don't want to change the recipe a lot)

**Comment by Ayutac** [comment]
that is a problem for the modpack creator, not for us
